website/docs/r/edgegateway_settings.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_edgegateway_settings"
+sidebar_current: "docs-vcd-resource-edgegateway-settings"
+description: |-
+  Provides a vCloud Director edge gateway global settings. This can be used to update global edge gateways settings related to firewall and load balancing.
+---
+
+# vcd\_edgegateway\_settings
+
+Provides a resource that can update vCloud Director edge gateway global settings either as system administrator or as
+organization user.
+
+The main use case of this resource is to allow both providers and tenants to change edge gateways global settings (such as
+enabling load balancing or firewall) when the edge gateway was created outside of terraform.
+
+~> **Warning:** The edge gateway settings info is tied to an edge gateway. Thus, there could be only one instance per 
+edge gateway. Using a different definition for the same edge gateway ID will result in a previous instance to be overwritten.
+
+Supported in provider *v3.0+*
+
+## Example Usage
+
+```hcl
+data "vcd_edgegateway" "egw" {
+  name = "my-egw"
+}
+
+resource "vcd_edgegateway_settings" "egw-settings" {
+  edge_gateway_id         = data.vcd_edgegateway.egw.id
+  lb_enabled              = true
+  lb_acceleration_enabled = true
+  lb_logging_enabled      = true
+  lb_loglevel             = "debug"
+
+  fw_enabled                      = true
+  fw_default_rule_logging_enabled = true
+  fw_default_rule_action          = "accept"
+}
+```
+
+-> **Tip:** Although this resource changes values in the edge gateway referenced as a data source, Terraform state
```

Comment:
This is not our problem bu by design how Terraform works (datasources take precedence unless they have references to some computed fields) so not sure if we need to detail it to look as our fault. Up to you though.

Reply:
I don't want to point at it as a fault, but I need users to be aware of it.
I could change it to 
```
-> **Tip:** Although this resource changes values in the edge gateway referenced as a data source, due to how Terraform works, the state
```

Reply:
Works for me.